extension/NotificationService.swift

```diff
@@ -0,0 +1,12 @@
+import BareKitBridge
```

Comment:
This is the bad guy! Impossible to use `BareKit` in that which expose only a `BareKit.IPC` and `BareKit.Worklet`.

Still don't know why :)

Reply:
I am on it!

Reply:
This PR should fix this. https://github.com/holepunchto/bare-kit-swift/pull/1/files